src/model/user.ts

```diff
@@ -1,10 +1,13 @@
-import { ActorType } from "activitypub-objects";
+import { ActorType, Actor } from "activitypub-objects";
 
-export interface User {
+export interface User extends Actor {
```

Comment:
Maybe:

```suggestion
export interface InternalUser extends Actor {
```

Reply:
What do you think about:

```suggestion
export interface InternalActor extends Actor {
```

Reply:
Good for me as well. :)

Reply:
@fgervasi from what I see, this is  the only comment that's left. After that we can merge this branch into master! 🥳